h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoostUtils.java

```diff
@@ -134,6 +140,29 @@ else if (f.vec(i-catCols).isInt())
         return trainMat;
     }
 
+    // FIXME this and the other method should subtract rows where response is 0
```

Comment:
Should this be fixed before the PR is merged?

Reply:
would be nice but there are probably workarounds for this one (just removing such rows from the dataset)